test/helpers/kubectl.go

```diff
@@ -1129,14 +1129,17 @@ func (kub *Kubectl) CiliumEndpointWaitReady() error {
 		return true, nil
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), HelperTimeoutDuration)
+	ctx, cancel := context.WithTimeout(context.Background(), HelperTimeout)
 	defer cancel()
 	err = WithContext(ctx, body, 1*time.Second)
 	if err == nil {
 		return err
 	}
 
 	callback := func() string {
+		ctx, cancel := context.WithTimeout(context.Background(), HelperTimeout)
+		defer cancel()
```

Comment:
I think these lines were actually introduced in another commit, but this matches master so LGTM

(Found the commit later, it actually belongs to `helpers: Limit ep gathering to 5m for all cilium pods`)